### PR TITLE
fix: Show 4 identical colors instead of 1

### PR DESCRIPTION
**Pull request by roberthgnz**
Related to #5

### CODE REVIEW

File: api/generate.js [hunk 1]
```diff
@@ -32,8 +32,15 @@ const generate = async (req, res) => {
   const mainColors = predominant.length > 4 ? predominant.slice(0, 4) : predominant;
 
   if (colors) {
+    const names = {}
     return res.json({
-      colors: mainColors.map(color => {
+      colors: mainColors.filter(color => {
+        const name = colorNamer(color).ntc[0].name
+        if(!names[name]) {
+          names[name] = true
```
Comment (mayashavin) [inline api/generate.js]: Isn't it better to just use `reduce` to create mapping for distinct colors and then use Object.values() to export that new array where there is only distinct color?

For example:

```
const colors = mainColors.reduce((filteredColors, currColor) => {
  if (filteredColor[colorNamer(color).ntc[0].name]) return filteredColors;

   const main = {
          name: colorNamer(color).ntc[0].name,
          hex: color,
          text: getTextColor(color)
        };

        const palettes = generatePalettes(main);
        
        filteredColors[colorNamer(color).ntc[0].name] =  {
          main,
          palettes,
          code: getCodes(main, palettes)
        }
         
  return filterColors
}, {})

return res.json({
  colors: Object.values(colors),
//other code
})
```

WDYT?

File: pages/index.vue [hunk 2]
```diff
@@ -70,7 +70,7 @@
         <div
           v-for="color in demo.colors"
           :key="color.main.hex"          
-          :class="`md:w-1/${demo.colors.length}`"
+          :class="`${demo.colors.length > 1 ? `md:w-1/${demo.colors.length}` : 'md:w-full'}`"
```
Comment (mayashavin) [inline pages/index.vue]:
```
:class="`md:${demo.colors.length > 1 ? `w-1/${demo.colors.length}` : 'w-full'}`"

```

**Review by mayashavin — CHANGES_REQUESTED** [review]

File: api/generate.js (hunk 1 above)
Comment (roberthgnz) [reply api/generate.js]: You are right. It's a better solution.

**Review by mayashavin — APPROVED** [review]

**Comment by mayashavin** [comment]
Thank you @eliutdev for your contribution :)

**Pull request merged**